docs/CNAME

```diff
@@ -0,0 +1 @@
+developers.trustroots.org
```

Comment:
FYI, since https://github.com/Trustroots/developers had the same CNAME, there was a conflict:

>[Trustroots/trustroots] Page build warning
>
> The page build completed successfully, but returned the following warning for the `master` branch:
> 
> The CNAME `developers.trustroots.org` is already taken. Check out https://help.github.com/articles/troubleshooting-custom-domains/#cname-already-taken for more information.
> 
> For information on troubleshooting Jekyll see:
> 
>  https://help.github.com/articles/troubleshooting-jekyll-builds
> 
> If you have any questions you can contact us by replying to this email.

Might require modifying something in this folder (or this CNAME file) again before https://developers.trustroots.org/ becomes visible again

Reply:
I moved `InfluxDB.md` into `docs/`, that didn't fix it.

Reply:
What about creating a `gh-pages` branch? Shouldn't this stuff be in there?

Reply:
It works from docks folder these days, too. That way there is no need to fiddle with different branches.

Reply:
did `git push --set-upstream origin gh-pages`, now waiting

Reply:
Hmm, I would not go that route; it would require to deploy changes to two different branches. Master branch should work:

https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch

I think we could just disable pages on this repo and enable them again to make github realise there is something to build here. 🙃